### PR TITLE
Buttons

### PR DESCRIPTION
**Pull request by anandsr21harshit**
Three types of buttons

1. Simple buttons
2. Outline buttons
3. Disabled buttons

### CODE REVIEW

File: Components/Buttons/Buttons.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+.container{
+    max-width: 600px;
+    border: 2px solid black;
```
Comment (Parul1999) [inline Components/Buttons/Buttons.css]: Use same units , e.g border-radius is in rem but border is in px . Consistency is needed.

**Review by Parul1999 — CHANGES_REQUESTED** [review]
Use the same units in CSS, as mentioned in the comment.

**Review by OmKakatkar — APPROVED** [review]
Looks good to merge, a suggestion would be to explore on the disabled attribute of the button element.

**Comment by anandsr21harshit** [comment]
Units changed

**Review by scjuly19 — APPROVED** [review]

**Pull request merged**